src/app/games/new-game/NewRulesConfig.ts

```diff
@@ -0,0 +1,33 @@
+import { RulesConfigDescription } from 'src/app/components/normal-component/pick-game/pick-game.component';
+import { MGPValidation } from 'src/app/utils/MGPValidation';
+
+// Note: RulesConfigDescriptions object are right now centralised in RulesConfigDescriptions
+export const newRulesConfigDescription: RulesConfigDescription =
```

Comment:
newGame* déjà
puis, est-ce que pour NewGame on veut pas garder un truc simple pour que des gens puissent découvrir comment coder un jeu ? Puis si ils souhaitent ajouter des config, là on peut dire "bah tu connais un peu le code MGP, va donc voir P4 pour inspiration !" Sinon ça peut être effrayant

Reply:
// It is FULLY mandatory, if you don't want to make your game configurable just yet, ignore this!